### PR TITLE
[setup,test][xl]: Set test environment

### PR DESCRIPTION
**Pull request by sglavoie**
This is for issue #8.

This may require some changes, I'm listening. What's currently missing:

* Integration with Travis CI.
* Integration with Code Climate.

I'm about to add comments to the code to ask a few questions. @hannelita, your help is welcome :wink:.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+language: python
+
+python:
+  - 3.7
+  - 3.8
+
+env:
+  global:
+    - CC_TEST_REPORTER_ID=a0e5783a230ba11a5eece9d4bac3960396b9f6c319e9326796f713bcc7030389
```
Comment (sglavoie) [inline .travis.yml:9]: I haven't searched yet what this ID is needed for.

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,60 @@
+# Makefile for AirCan
+PACKAGE_NAME := aircan
+PACKAGE_DIRS := aircan
+TESTS_DIR := tests
+VERSION_FILE := VERSION
+
+PYTEST_EXTRA_ARGS := --flake8 --isort --doctest-modules -rsxX -l --tb=short --strict --cov-report term-missing:skip-covered --cov=tests --cov-fail-under 100
```
Comment (sglavoie) [inline Makefile:7]: Should we maybe stick to a simpler `--flake8 --isort --doctest-modules` without the remaining options? It makes for a nice output, but might be cumbersome if many tests fail at once.

File: pytest.ini [hunk 3]
```diff
@@ -0,0 +1,5 @@
+[pytest]
+addopts = --flake8 --isort --doctest-modules -rsxX -l --tb=short --strict --cov-report term-missing:skip-covered --cov=tests --cov-fail-under 100
```
Comment (sglavoie) [inline pytest.ini:2]: Same here, there might be a few extra options we may want to get rid of.

File: setup.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+from setuptools import setup
+
+setup(
+    name='AirCan',
+    packages=['aircan'],
+    version=open('VERSION').read(),
+    description='',
+    author='Datopian',
+    author_email='info@datopian.com',
```
Comment (sglavoie) [inline setup.py:9]: In the Giftless repo, this was set to the email of the original committer at Datopian. I'm not sure what's the best option here.

File: tests/data_loader_config.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
+import os
```
Comment (sglavoie) [inline tests/data_loader_config.py:1]: Note: This file is being added to make the tests run. Is this OK?

File: tests/data_loader_load.py [hunk 6]
```diff
@@ -0,0 +1,208 @@
+import hashlib
```
Comment (sglavoie) [inline tests/data_loader_load.py:1]: Note: This file is being added to make the tests run. Is this OK (again)?

File: tests/test_load.py [hunk 7]
```diff
@@ -1,9 +1,9 @@
-import os
 import unittest
 
 from sqlalchemy import create_engine
 
-from data_loader import load, config
+import data_loader_config as config
```
Comment (sglavoie) [inline tests/test_load.py:5]: Those imports will not be needed if we change the tests.

File: tests/data_loader_config.py (hunk 5 above)
Comment (hannelita) [reply tests/data_loader_config.py:1]: Yes, I think it's fine for now

**Comment by hannelita** [comment]
@sglavoie Great work! I'll merge it for now; I think CI integration deserves a separate PR.

**Pull request merged**